HowToRPC.md

```diff
@@ -0,0 +1,64 @@
+# How to Call Google APIs: RPC Edition
+
+_For many developers, the easiest way to call a Google API is with one of our client libraries. But occasionally someone may prefer to make API calls directly - perhaps from a language or environment that we don’t support or using a different networking library or tool. Here we’ll show you how to do it._
+
+This page focuses on calling Google APIs directly using their underlying RPC interfaces. Most of these APIs are also available as REST services. For that, see [How to Call Google APIs, REST Edition](/HowToREST).
+
+## What you’ll need
+
+### An API definition
+Most Google APIs are designed as [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call) APIs using the [Protocol Buffers language](https://developers.google.com/protocol-buffers/docs/overview). The Protocol Buffers definitions of public Google APIs are hosted on GitHub in the [googleapis/googleapis](https://github.com/googleapis/googleapis) repository.
+
+For these examples, we’ll use the [Cloud Natural Language API](https://cloud.google.com/natural-language/), which is defined by [google/cloud/language/v1/language_service.proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/language/v1/language_service.proto). The RPC details are documented online in the [Google Cloud Natural Language API reference](https://cloud.google.com/natural-language/docs/reference/rpc/). We’ll call the [AnalyzeEntities](https://cloud.google.com/natural-language/docs/reference/rpc/google.cloud.language.v1#google.cloud.language.v1.LanguageService.AnalyzeEntities) API, which takes a block of text as input and returns a list of names and nouns that it finds in the text along with some interesting properties of each entity.
+
+### Protocol Buffers 
+For all Google RPC APIs, the messages that are sent and received using the [Protocol Buffers](https://developers.google.com/protocol-buffers/docs/overview) encoding, and the definitive descriptions of these APIs are written in the Protocol Buffers Language. 
+
+To compile Protocol Buffer Language files, you’ll need `protoc`, the Protocol Buffer compiler. You can download `protoc` from the [google/protobuf release page](https://github.com/google/protobuf/releases) on GitHub or build it from source. You’ll probably also need a code generation plugin for the language that you’re using. Plugins are standalone executables written in many different languages, and the plugin interface is defined in the [plugin.proto](https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/plugin.proto) file. Here are some plugins that we have used:
+
+- **Go** [github.com/golang/protobuf](https://github.com/golang/protobuf)
```

Comment:
Idea: Maybe we should link to the wiki page listing add-ons rather than have yet another mini list?
(I am not sure if this is actually what we should do, proposing for discussion.)

Reply:
I think initially it would be simple to keep all this together, but let's see how big this page gets.